internal/controller/drplacementcontrol.go

```diff
@@ -1211,20 +1245,25 @@ func (d *DRPCInstance) isVRGConditionMet(cluster string, conditionType string) b
 	if vrg == nil {
 		d.log.Info(fmt.Sprintf("isVRGConditionMet: VRG not available on cluster %s", cluster))
 
-		return !ready
+		return !ready, fmt.Errorf("VRG not available on cluster %s", cluster)
```

Comment:
If add an error, we can remove the info log. With the error we will this issue twice. Once here and once in the top level function handling the error we return here.

Same issue for in the next changes.

Reply:
Done